block-layer/codecs/DAG-JSON.md

```diff
@@ -24,14 +24,18 @@ Contrary to popular belief, JSON as a format supports Big Integers. It's only
 JavaScript itself that has trouble with them. This means JS implementations
 of `dag-json` can't use the native JSON parser and serializer.
 
+### Other Types
+
+The IPLD Data Model specifies two types that cannot directly be represented in JSON. They are both represented as [Multibase](https://github.com/multiformats/multibase) Base64 encoded strings. The Base64 encoding is the one described in [RFC 4648, section 4](https://tools.ietf.org/html/rfc4648#section-4) without padding, hence the Multibase prefix is `m`.
+
 #### Binary Type
 
 ```javascript
-{"/": { "base64": String }}
+{"/": { "base64": String /* Multibase Base64 encoded binary */ }}
```

Comment:
isn't the `"base64"` key redundant if you are going to store the base64 prefix in the string? 

If this key is unneeded maybe it should be more compact? Seems like adding the multibase prefix is breaking anyway, so why not update that when doing this?

Reply:
We still need some kind of key, because it can’t be a bare string as we’re already using that for the `Link` type

Reply:
Yeah, my suggestion was just for maybe something more compact since there is no new information given with this key.

Reply:
"bytes"? Or "multibase"?

Reply:
I also think "base64" isn't such a good name. I like @Stebalien proposition of "bytes", it aligns well with the naming the [IPLD Data Model](https://github.com/ipld/specs/blob/2014318ecba05c3a506d2676d49385905f86d7c4/data-model-layer/data-model.md#kinds).

So it would read:

```javascript
{"/": { "bytes": String /* Multibase Base64 encoded binary */ }}
```